serverless/knative_eventing/serverless-using-brokers.adoc

```diff
@@ -6,11 +6,44 @@ include::modules/serverless-document-attributes.adoc[]
 
 toc::[]
 
-Knative Eventing uses the `default` broker unless otherwise specified.
+Knative provides a multi-tenant (MT) Broker implementation that uses Channels for event routing.
+An MT Broker instance can be created by both users and cluster administrators.
 
-If you have cluster administrator permissions, you can create the `default` broker automatically using namespace annotation.
+== Creating the MT broker
 
-All other users must create a broker using the manual process as described in this guide.
+* Create the Broker:
++
+[source,yaml]
+----
+$ oc create -f - <<EOF
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    eventing.knative.dev/broker.class: MTChannelBasedBroker
+  name: default
+  namespace: default
+spec:
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: config-br-default-channel <1>
+    namespace: knative-eventing
+EOF
+----
 
-include::modules/serverless-creating-broker.adoc[leveloffset=+1]
-include::modules/serverless-creating-broker-admin.adoc[leveloffset=+1]
+<1> The name of the ConfigMap used by the Channel you want to connect to the Broker. For more information, see xref:../../serverless/knative_eventing/serverless-channels.adoc#serverless-channels[Using Channels]
+
+== Creating the `default` Broker
+
+[IMPORTANT]
+====
+For Knative versions 0.15 and earlier, the `default` Broker is used.
```

Comment:
I think we should stick to Serverless versioning. For example for 1.8.0-rc1 the MT broker was a default one already, despite that 1.8.0-rc1 was shipping with Knative 0.14.x.

I'd say that for Serverless 1.7.x the `default` old broker is the default. Next Serverless release is probably be shipped with MT broker by default.